cloudify_aws/common/decorators.py

```diff
@@ -850,7 +853,13 @@ def wrapper(**kwargs):
             ctx.node.properties.get('Tags'),
             ctx.instance.runtime_properties.get('Tags'),
             kwargs.get('Tags'))
-        if iface and tags and resource_ids:
+        if isinstance(iface, (ELBBase, EKSBase)):
+            can_be_deleted = True
+        else:
+            cfg_copy = deepcopy(kwargs['resource_config'])
+            cfg_copy['DryRun'] = True
```

Comment:
We don't need this anymore:

            cfg_copy = deepcopy(kwargs['resource_config'])
            cfg_copy['DryRun'] = True